python/Ganga/Runtime/Repository_runtime.py

```diff
@@ -134,7 +134,15 @@ def bootstrap():
         registry.startup()
         logger.debug("started " + registry.info(full=False))
         if registry.name == "prep":
-            registry.print_other_sessions()
+            # Get a list of strings describing other running sessions
+            other_sessions = registry.get_other_sessions()
```

Comment:
Frankly leaving it in the prep registry check was for a minimal change. We want to disable the monitoring at a higher level than an individual repo init imo incase we ever want to do something such as drop the prepared state repo.
If we're going to start accessing the repo and subverting the registry than I'll move for that option, but I still think this is bad practice.